### DISCUSSION TITLE
Github setup directions appear outdated.

### DISCUSSION THREAD

**Discussion by lockerk23** [community/community#82724] [New to GitHub]
Watching Development Environment Setup Part 3, Github and Browser Sync.  Created repository and tried to "Launch auto. page generator" but it's not there.  What are the updated directions to do this and other parts of this module?

**Comment by varunpats** [comment]
Hey @lockerk23,

Hope this is this related to gihub pages. Please check out this [video tutorial](https://www.youtube.com/watch?v=QyFcl_Fba-k) on gh pages.

If you have any further doubt, please continue in comments.
